index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+	<html>
+	<head>
+		<link rel="stylesheet" type="text/css" href="css/styles.css">
+		<title>KANYE RESUME</title>
+	</head>
+
+	<body>
+		<nav></nav>
+		<nav></nav>
+			
+			<div class="header">
+				<h1>KANYE WEST</h1>
+
+				<ul>
+					<li>
+						<a href="quotes.html">QUOTES</a>
+					</li>
+					<li style="color: black"><b>||</b></li>
+					<li style="color: black">ME, MYSELF & I</li> 
+				</ul>
+			</div>
+		
+		<nav></nav>
+
+			<div class="left">
+				<h2>I AM GOD</h2>
+				
+				<h2>I AM KING</h2>
+				
+				<h3>I AM YEEZUS</h3>
+				
+				<h2>...I AM AN ONLY CHILD</h2>
+
+				<p>Things you may not know about me:</p>
+				<p>I have a metal plate in my chin from a near fatal car accident that happened on October 23, 2002. I was driving home from a studio session and fell asleep behind the steering wheel and crashed into another car.</p>
+				<p>I can't speak Chinese.</p>
+				<p>I like to stare at myself in the mirror.</p>				
+			</div>
+		
+			<div class="right">
+				<img src="images/kanye-kiss.png"/>	
+			</div>
+
+		<nav></nav>
+
+			<div class="footer">
+				<ul>
+					<li>
+						<a href="iamgod.html" target="_blank">
+							<img src="images/instagramA.svg" alt="instagram" style="height: 50px" width="50px" >
```

Comment:
:hammer: As mentioned before, avoid using inline styling. Add a class to the `<img>` and style it with css instead.